core/gesture.js

```diff
@@ -841,6 +842,9 @@ Blockly.Gesture.prototype.duplicateOnDrag_ = function() {
     var xy = this.targetBlock_.getRelativeToSurfaceXY();
     newBlock.moveBy(xy.x, xy.y);
     newBlock.setShadow(false);
+    // Preserve the label when copying
+    //newBlock.getField("VALUE").setValue(this.targetBlock_.getFieldValue("VALUE"));
+    //newBlock.getField("VALUE").EDITABLE = true;
```

Comment:
Commented out and left in unintentionally?

Reply:
Unintentional.  I'll pull it out and then merge after that.